datahub/config.yaml

```diff
@@ -33,6 +34,14 @@ hub:
     # HACK: manually edited deployment to add volume to /srv/log
     c.JupyterHub.extra_log_file = '/srv/log/jupyterhub.log'
 
+    if 'HUB_USERNAME_MAP' in os.environ:
```

Comment:
Add a note saying this is a temporary fix that should go away soon?

Reply:
Done!